### ISSUE TITLE
Hyrax - Check that RIIIF requests comply with access control

### ISSUE THREAD

**Issue by cudevmaxwell** [cu-library/hyrax-ansible#32]
For example, if an image is locked down to institutional or private access, the image should not be available through RIIIF to the public.

**Comment by cudevmaxwell** [comment]
I'd like confirmation from @orangewolf that this is resolved. I'll also double check.

**Comment by orangewolf** [comment]
@cudevmaxwell yes, a double check is always a good idea. I know the product owner for Hyku (some bloke named @orangewolf) and we'd consider it a pretty big bug if permissions were not checked in RIIIF ;-)

**Comment by orangewolf** [comment]
I've confirmed this by trying to access private images by accessing their full path. Neither manifests or images can be accessed
